webdriver/tests/execute_script/execute.py

```diff
@@ -70,3 +87,26 @@ def test_abort_by_user_prompt_twice(session, dialog_type):
     assert session.alert.text == "Bye"
 
     session.alert.accept()
+
+
+def test_execute_pinned(session):
+    pin_script(session, "One", "return 1;")
+    response = execute_script(session, name="One")
+    assert_success(response, 1)
+
+
+def test_name_and_script(session):
+    pin_script(session, "One", "return 1;")
+    response = execute_script(session, "return 1;", name="One")
+    assert_error(response, "invalid argument")
+
```

Comment:
The spec prose says the script needs to be a string but there are no tests proving what happens if you pass in something else

Reply:
https://github.com/web-platform-tests/wpt/pull/19675/files#diff-4a0029ee9b4392c27f3794abe77949faR40